lib/c-api/build.rs

```diff
@@ -89,8 +87,41 @@ fn build_wasm_c_api_headers(crate_dir: &str, out_dir: &str) {
     out_header_file.push("wasmer_wasm");
 
     let mut pre_header = format!(
-        r#"// The Wasmer C/C++ header file compatible with the `wasm-c-api` standard API.
-// This file is generated by lib/c-api/build.rs.
+        r#"// The Wasmer C/C++ header file compatible with the [`wasm-c-api`]
+// standard API, as `wasm.h` (included here).
+//
+// This file is automatically generated by `lib/c-api/build.rs` of the
+// [`wasmer-c-api`] Rust crate.
+//
+// # Stability
+//
+// The [`wasm-c-api`] standard API is a _living_ standard. There is no
+// commitment for stability yet. We (Wasmer) will try our best to keep
+// backward compatibility as much as possible. Nonetheless, some
+// necessary API aren't yet standardized, and as such, we provide a
```

Comment:
Small typo, should be `APIs`, I'm not going to review the grammar too closely, that kind of thing is trivial to fix later